xpdacq/utils.py

```diff
@@ -372,18 +360,26 @@ def create_yaml(self, bt):
             parsed_sa_md_list.append(parsed_sa_md)
         self.parsed_sa_md_list = parsed_sa_md_list
 
+
+    def create_yaml(self, bt):
+        """create xpdacq.Sample objects based on parsed metadata dict
+
+        Parameters
+        ----------
+        bt : xpdacq.Beamtime object
+            an object carries SAF, PI_last and other information
+
+        Returns
+        -------
+        None
+        """
+
         # normal sample, just create
-        for el in self.parsed_sa_md_list:
-            Sample(bt, el)
-        # separate so that bkg is in the back
-        bkg_name_list = [el['sample_name']
-                         for el in self.parsed_sa_md_list if
-                         el['sample_name'].startswith('bkgd')]
-        for bkg_name in set(bkg_name_list):
-            bkg_dict = {'sample_name': bkg_name,
-                        'sample_composition': {bkg_name: 1},
-                        'is_background': True}
-            Sample(bt, bkg_dict)
+        for md_dict in self.parsed_sa_md_list:
+            sample_name = md_dict.get('sample_name')
+            if sample_name.startswith('bkgd'):
```

Comment:
Is there a better way to be more explicit about this? (eg, have a field in the spreadsheet stating that a sample is a background?)